_content/canon/connection-with-pevious-deeds_anandajoti.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Pubbakammapilotika-Buddhāpadānaṁ: The Traditions about the Buddha (known as)
```

Comment:
For canonical works on SuttaCentral we use the SC ID. In this case [`tha-ap392`](https://suttacentral.net/tha-ap392/en/walters?lang=en&reference=none&highlight=false) In this case `+cmy` because it includes the canonical commentary: so the file name should be `tha-ap392+cmy.md`